src/types/apiErrorsType.ts

```diff
@@ -0,0 +1,5 @@
+export type EmptyInputErrorType = 'REQUIRED';
+
+export const emptyInputError: EmptyInputErrorType = 'REQUIRED';
```

Comment:
It's too much, if you set some string it gets only this string as type so you don't need to create another type for constant